### ISSUE TITLE
[FIX] Replace Frame.append with pd.concat

### ISSUE THREAD

**Issue by kmaphoenix** [GoogleCloudPlatform/dfcx-scrapi#35]
Context:
```
/layers/google.python.pip/pip/lib/python3.9/site-packages/dfcx_scrapi/core/conversation.py:117: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.
/layers/google.python.pip/pip/lib/python3.9/site-packages/dfcx_scrapi/core/conversation.py:117: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.
```

Reference:
https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/ee744d3fef9b0f1d53f6508dd1d91941731f8dc5/src/dfcx_scrapi/core/conversation.py#L117

**Comment by GowthamChowdary** [comment]
i'd like to work on this. i'd appreciate some clarification, do we want to replace frame.append in every instance? or just in coversations.py?

**Comment by kmaphoenix** [comment]
> do we want to replace frame.append in every instance? or just in coversations.py?

For now we can start in just `conversations.py`.  
I suspect there may be other instances of this in other classes as well, but we can assess them as we go through cleanup and refactoring.

**Comment by GowthamChowdary** [comment]
On it. Thanks.

**Comment by GowthamChowdary** [comment]
On it . Thanks

**Comment by kmaphoenix** [comment]
Fixed in PR #52
